cmd/config.go

```diff
@@ -488,24 +488,26 @@ func DefaultConfig(memo string) *Config {
 
 // GlobalConfig describes any global relayer settings
 type GlobalConfig struct {
-	APIListenPort   string `yaml:"api-listen-addr" json:"api-listen-addr"`
-	Timeout         string `yaml:"timeout" json:"timeout"`
-	Memo            string `yaml:"memo" json:"memo"`
-	LightCacheSize  int    `yaml:"light-cache-size" json:"light-cache-size"`
-	LogLevel        string `yaml:"log-level" json:"log-level"`
-	ICS20MemoLimit  int    `yaml:"ics20-memo-limit" json:"ics20-memo-limit"`
-	MaxReceiverSize int    `yaml:"max-receiver-size" json:"max-receiver-size"`
+	APIListenPort     string `yaml:"api-listen-addr" json:"api-listen-addr"`
```

Comment:
Is this `APIListenPort` used anywhere else other than for the debug server anymore?

Reply:
APIListenPort is only used for debug server

Reply:
perhaps we should rename this to `DebugListenAddr` now as well so it's more clear for users

Reply:
renamed and added some tests